generator/src/generator/error_events.rs

```diff
@@ -490,6 +490,42 @@ fn generate_events(out: &mut Output, module: &xcbgen::defs::Module) {
         outln!(out, "}}");
         outln!(out, "");
 
+        outln!(
+            out,
+            "/// Get the additional length if any (in 4 byte units) of this X11 event.",
```

Comment:
```suggestion
            "/// Get the additional length beyond 32 bytes if any (in 4 byte units) of this X11 event.",
```
(I'm not quite happy with the wording of my proposal; suggestions welcome. What I want to propose is to mention the "base size" of 32 bytes somehow.)